File: Uri-Online-Judge/Wellington/Python/Iniciante/1036.py
# -*- coding: utf-8 -*-

import math

linha = raw_input().split(' ') #ler valores
a, b, c = linha #array para variaveis
a = float(a) #converte valores
b = float(b)
c = float(c)

delta = pow(b, 2) - 4 * a * c #calcula delta

if a != 0 and delta > 0: #condição para calculo
	r1 = (-b + math.sqrt(delta)) / (2 * a)
	r2 = (-b - math.sqrt(delta)) / (2 * a)

	print('R1 = %.5f' % r1)
	print('R2 = %.5f' % r2)

else: #não pode ser calculado 
	print('Impossivel calcular')

File: Uri-Online-Judge/Wellington/C:C++/Iniciante/1036.cpp
#include <stdio.h>
#include <math.h>

int main(){
    double a = 0, b = 0, c = 0, r1, r2, delta;
    
    scanf("%lf", &a);
    scanf("%lf", &b);
    scanf("%lf", &c);
    
    if (a != 0 && (pow(b,2) - 4*a*c) > 0){
        delta = pow(b,2) - 4 * a * c;
        r1 = (-b + sqrt(delta)) / (2 * a);
        r2 = (-b - sqrt(delta)) / (2 * a);
        
        printf("R1 = %.5lf\n", r1);
        printf("R2 = %.5lf\n", r2);
    } else {
        printf("Impossivel calcular\n");
    }
    
    return 0;
}
